integration-core/pom.xml

```diff
@@ -17,5 +17,13 @@
             <artifactId>integration-healthcheck</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.symphonyoss</groupId>
+            <artifactId>integration-webhook</artifactId>
+            <version>${project.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Seems weird to have core module depend on webhook module, even for test scope/jar.  Might create a circular dependency.